Tests/cryptography/NetflixVaultGenerator/main.cpp

```diff
@@ -0,0 +1,130 @@
+#include <core/core.h>
+#include <openssl/bio.h>
+#include <openssl/evp.h>
+#include <openssl/aes.h>
+#include <iostream>
+#include <fstream>
+#include <openssl/modes.h>
+
+using namespace std;
+
+const uint32_t g_esnSize = 42;
```

Comment:
Why is the ESN size fixed? I don't think it's always 42...? Did a quick search on the net and found different sizes of sample ESNs. The TA even has dedicated API to retrieve the ESN size, hence my doubts.